common/analysis/processingChainController.js

```diff
@@ -0,0 +1,43 @@
+import * as Chain from './processingChain'
+import * as Step from './processingStep'
+import * as StepVariable from './processingStepVariable'
+import * as Calculation from './processingStepCalculation'
+
+export const assocCalculation = ({ chain, step, calculation }) => {
```

Comment:
create a single file per method and export them from index.js

Reply:
ok

Reply:
ok